scripts/flash/designated-report.js

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env node
+
+"use strict";
+
+var chalk = require("chalk");
+var getTime = require("./get-timestamp");
+
+/**
+ * Move time to Market end time and do initial report
+ */
+function designateReportInternal(augur, marketId, outcome, invalid, auth) {
```

Comment:
Nitpick: suggest using `marketID` instead of `marketId` for consistency, and pick one of `outcome` or `outcomeID` (not `outcomeId`) but do not use both `outcome` and `outcomeId` to refer to the same thing.

Reply:
Will do, we are moving to `Id` in UI, I'll change to `ID` for augur.js

Reply:
If you're using `Id` in the UI, that's fine to use it here, too.